src/browser/BLECentralPlugin.js

```diff
@@ -1,63 +1,194 @@
+            }).then(result => {
+                success(result);
+            }).catch(error => {
+                if (failure) failure(error);
+            });
+        } else if (failure) { 
+          failure();
+        }
     },
     writeWithoutResponse: function(device_id, service_uuid, characteristic_uuid, data, success, failure) {
-        notSupported();
-        if (failure) failure();
+        if (this.devices.has(device_id)) {
+            this.devices.get(device_id).server.getPrimaryService(service_uuid).then(service => {
+                return service.getCharacteristic(characteristic_uuid);
+            }).then(characteristic => {
+                return characteristic.writeValue(data);
+            }).then(result => {
+                success(result);
+            }).catch(error => {
+                // Ignore Error 
+                // failure();
+            });
+        } else if (failure) { 
+            // Ignore Error 
+            // failure();
+        }
     },
     startNotification: function(device_id, service_uuid, characteristic_uuid, success, failure) {
-        notSupported();
-        if (failure) failure();
+         if (this.devices.has(device_id)) {
+            this.devices.get(device_id).server.getPrimaryService(service_uuid).then(service => {
+                return service.getCharacteristic(characteristic_uuid);
+            }).then(characteristic => {
+                return characteristic.startNotifications().then(result => {
+                    characteristic.addEventListener('characteristicvaluechanged', function (event) {
+                        success(event.target.value.buffer);
+                    });
+                });
+            }).catch(error => {
+                if (failure) failure(error);
+            })
+        } else if (failure) { 
+          failure();
+        }
     },
     stopNotifcation: function(device_id, service_uuid, characteristic_uuid, success, failure) {
-        notSupported();
-        if (failure) failure();
+       if (this.devices.has(device_id)) {
+            this.devices.get(device_id).server.getPrimaryService(service_uuid).then(service => {
+                return service.getCharacteristic(characteristic_uuid);
+            }).then(characteristic => {
+                return characteristic.stopNotifications();
+            }).then(result => {
+                success(result);
+            }).catch(error => {
+                if (failure) failure(error);
+            });
+        } else if (failure) { 
+          failure();
+        }
     },
     isEnabled: function(success, failure) {
         notSupported();
         if (failure) failure();
     },
     isConnected: function(device_id, success, failure) {
-        notSupported();
-        if (failure) failure();
+        if (this.devices.has(device_id)) {
+            var device = this.devices.get(device_id).device;
+            if (device.gatt.connected) {
+                device.gatt.disconnect();
```

Comment:
This method should simply be returning whether the device gatt is connected. `success` if connected, or `failure` for all other cases... definitely shouldn't be disconnecting the device though 😅